lib/Sema/TypeChecker.cpp

```diff
@@ -363,7 +363,8 @@ static void bindExtensionDecl(ExtensionDecl *ED, TypeChecker &TC) {
   // FIXME: Perform some kind of "shallow" validation here?
   TypeResolutionOptions options(TypeResolverContext::ExtensionBinding);
   options |= TypeResolutionFlags::AllowUnboundGenerics;
-  if (TC.validateType(ED->getExtendedTypeLoc(), dc, options)) {
+  if (TC.validateType(ED->getExtendedTypeLoc(),
+                      TypeResolution::forContextual(dc), options)) {
```

Comment:
And again

Reply:
I was assuming you'd do this one as part of https://github.com/apple/swift/pull/18661 ?